src/domains/sign/io_sign_support_func.tf

```diff
@@ -0,0 +1,186 @@
+locals {
+  io_sign_support_func = {
+    app_settings = {
+      FUNCTIONS_WORKER_RUNTIME                        = "node"
+      WEBSITE_VNET_ROUTE_ALL                          = "1"
+      WEBSITE_ADD_SITENAME_BINDINGS_IN_APPHOST_CONFIG = "1"
+      WEBSITE_DNS_SERVER                              = "168.63.129.16"
+      WEBSITE_RUN_FROM_PACKAGE                        = "1"
+      FUNCTIONS_WORKER_PROCESS_COUNT                  = 4
+      AzureWebJobsDisableHomepage                     = "true"
+      NODE_ENV                                        = "production"
+      CosmosDbConnectionString                        = module.cosmosdb_account.connection_strings[0]
+      CosmosDbIssuerDatabaseName                      = module.cosmosdb_sql_database_issuer.name
+      CosmosDbUserDatabaseName                        = module.cosmosdb_sql_database_user.name
+      PdvTokenizerApiBasePath                         = "https://api.uat.tokenizer.pdv.pagopa.it"
+      PdvTokenizerApiKey                              = module.key_vault_secrets.values["TokenizerApiSubscriptionKey"].value
+    }
+  }
+}
+
+module "io_sign_support_func" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//function_app?ref=v4.1.6"
+
+  name                = format("%s-support-func", local.project)
+  location            = azurerm_resource_group.backend_rg.location
+  resource_group_name = azurerm_resource_group.backend_rg.name
+
+  health_check_path = "/api/v1/sign/support/info"
+
+  os_type          = "linux"
+  runtime_version  = "~4"
+  always_on        = true
+  linux_fx_version = "NODE|18"
+
+  app_service_plan_info = {
+    kind                         = "Linux"
+    sku_tier                     = var.io_sign_support_func.sku_tier
+    sku_size                     = var.io_sign_support_func.sku_size
+    maximum_elastic_worker_count = 0
+  }
+
+  app_settings = local.io_sign_support_func.app_settings
```

Comment:
Just as advice, I would keep the `app_settings` property structured as in https://github.com/pagopa/io-infra/blob/a77c3530f1312e553d661c9e7722d4f296a9f2f2/src/domains/sign/io_sign_user_func.tf#L107-L115
so that the IaC is ready in case we would add function triggers different that `HTTP`

Reply:
Using the `v6.0.1` we could make a step further: remove the comment and leverage `sticky_settings` property